tests/JakubOnderka/PhpVarDumpCheck/CustomFunctionTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
```

Comment:
Please, why isn't this commit included in implementation? I mean that will be better for people who read Git history :)